monadic-html/src/main/scala/scala/xml/xml.scala

```diff
@@ -9,21 +9,27 @@ import language.higherKinds
 // directly in scalac paser). `NoteSeq` disappeared, `Node <:!< Seq[Node]`...
 
 /** Trait representing XML tree. */
-sealed trait Node
+sealed trait Node {
+  def scope: Option[Scope] = None
+  def prefix: Option[String] = None
+  def namespace: Option[String] = if (prefix.isDefined) prefix.flatMap(namespaceURI) else scope.flatMap(_.namespaceURI(null))
+  def namespaceURI(prefix: String): Option[String] = scope.flatMap(_.namespaceURI(prefix))
```

Comment:
I think this could be removed after https://github.com/OlivierBlanvillain/monadic-html/pull/36/commits/f5421a31f65dc112da95f9a12a10c43aa5db1617#r98707126